k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -733,6 +778,40 @@ function deployCortxConfigMap()
         --from-file="${auto_gen_ha_path}"
 }
 
+function pwgen()
+{
+    # This function generates a random password that is
+    # 16 characters long, starts with an alphanumeric
+    # character, and contains at least one character each
+    # of upper case, lower case, digit, and a special character.
+
+    function choose()
+    {
+        seqlen=$1
+        charset=$2
+        # https://unix.stackexchange.com/a/230676
+        tr -dc "${charset}" < /dev/urandom | head -c "${seqlen}"
+    }
+
+    # Choose an alphanumeric char for the first char of the password
+    local first
+    first=$(choose 1 '[:alnum:]')
+
+    # Choose one of each of the required fields, plus 11 more
+    # characters, and shuffle them
+    local rest
+    rest=$({
+        choose 1 '!@#$%^'
+        choose 1 '[:digit:]'
+        choose 1 '[:lower:]'
+        choose 1 '[:upper:]'
+        choose 11 '[:alnum:]!@#$%^'
+    } | fold -w1 | shuf | tr -d '\n')
```

Comment:
I recognize that we need to do this based on some requirements of the components. Can we identify which components are requiring these, and submit issues? We should just be able to say "random 16 characters" and that's it.

Reply:
I'll raise a backlog issue on this (and I'll try to take care of it pretty quickly to clear it out)